### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by pimcoreanand** [pimcore/pimcore#16576] [Q&A / Support]
### Pimcore version

v11.1.4

### Steps to reproduce

Hi,

We have tried to sync OneDrive with Pimcore, but we could not sync whatever files and folders we created from OneDrive to the Pimcore admin panel dashboard. It is not reflecting in the admin panel. Please help me to achieve this.

Also, we can see files and folders on the server (pimcore project path) itself, but we cannot see them in the admin panel.

Kindly share the steps to achieve this.

How to sync OneDrive to Pimcore DAM.

Note: I have installed Pimcore version 11 on the Ubuntu server.

### Actual Behavior

Hi,

We have tried to sync OneDrive with Pimcore, but we could not sync whatever files and folders we created from OneDrive to the Pimcore admin panel dashboard. It is not reflecting in the admin panel. Please help me to achieve this.

Also, we can see files and folders on the server (pimcore project path) itself, but we cannot see them in the admin panel.

Kindly share the steps to achieve this.

How to sync OneDrive to Pimcore DAM.

Note: I have installed Pimcore version 11 on the Ubuntu server.

### Expected Behavior

How to sync OneDrive to Pimcore DAM.

**Comment by rushitkhunt1996** [comment]
Hi @pimcoreanand,

I think this reference will help you. clieck [here](https://packagist.org/packages/shitware-ltd/flysystem-msgraph)
For Pimcore configuration click [here](https://pimcore.com/docs/pimcore/10.6/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Storage_Setup.html)

**Reply by pimcoreanand** [reply]
Kindly share the details steps so it will be helpful
